stdlib/pushbullet/pushbullet.flux

```diff
@@ -0,0 +1,73 @@
+package pushbullet
+
+import "http"
+import "json"
+
+option defaultURL = "https://api.pushbullet.com/v2/pushes"
+
+// `pushData` sends a push notification using PushBullet's APIs.
+// `url` - string - URL of the PushBullet endpoint. Defaults to: "https://api.pushbullet.com/v2/pushes".
+// `token` - string - the api token string.  Defaults to: "".
+// `data` - object - The data to send to the endpoint. It will be encoded in JSON and sent to PushBullet's endpoint.
+// For how to structure data, see https://docs.pushbullet.com/#create-push.
+// NOTE: have to default to blank data to help type system figure out that should be an object.
+pushData = (url=defaultURL, token="", data={}) => {
```

Comment:
`data`'s default argument is causing a type error. The type system doesn't know how to unify an empty record `{}` with a nonempty record `{type: "note", title: title, body: text}`. Making `data` a required argument will fix this.

Reply:
I am a bit confused.

Tests pass locally but not in CI.

Tried to make `data` a required argument and I experience a failure.

🤔